config/database.yml

```diff
@@ -81,6 +81,7 @@ production:
     host: <%= IdentityConfig.store.database_socket.present? ?  IdentityConfig.store.database_socket : IdentityConfig.store.database_host %>
     password: <%= IdentityConfig.store.database_password %>
     pool: <%= primary_pool %>
+    prepared_statements: <%= !IdentityConfig.store.database_socket.present? %>
```

Comment:
I like this because it means less chance of operator error when configuring pgbouncer.

One idea could be to find a way to log non-sensitive parts of this config so that we could easily tell what's currently deployed

Reply:
@jgrevich mentioned similar idea, would be good to `Rails.logger` some of this somewhere